### ISSUE TITLE
What if we can't make scanner view full screen?

### ISSUE THREAD

**Issue by sainttail** [dm77/barcodescanner#91]
Then what about the scan view is in the fragment that also in base activity that will replace fragment (I cannot make scan qr fragment use full screen)

Here is my activity layout

```
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical">

    <include layout="@layout/actionbar_layout" />

    <FrameLayout
        android:id="@+id/scan_content"
        android:layout_width="match_parent"
        android:layout_height="0dp"
        android:layout_weight="1">

    </FrameLayout>

    <include layout="@layout/tabbar_layout" />
</LinearLayout>
```

scan content will be replace by scan fragment but any way it cannot stretch to fill full screen and it leave black black ground on left and right side

and apart from scan qr fragment I also have many other fragments that will be replaced here also.

**UPDATE:**
Can we somehow use the fullscreen data image from camera and crop only part that match our preview's size to show on preview's view  (this can also preserve ratio)

**Comment by xzwzyf0503** [comment]
I have the same problem that preview is not full screen, Do you have any solution.

**Comment by SudarAbisheck** [comment]
Just checkout this library too [ZXing Orient](https://github.com/SudarAbisheck/ZXing-Orient). It supports full screen and [Portrait Orientation](https://github.com/SudarAbisheck/ZXing-Orient) too.

**Comment by dm77** [comment]
Fixed in 1.9
